<launch>

   <rosparam file="$(find uta_pr2_forceControl)/default_pr2_controller.yaml"
             command="load" />

   <node pkg="pr2_controller_manager"
         type="spawner"
         args="r_arm_cartesian_pose_controller"
         name="r_arm_cartesian_pose_controller_spawner" />

   <node pkg="pr2_controller_manager"
         type="spawner"
         args="l_arm_cartesian_pose_controller"
         name="l_arm_cartesian_pose_controller_spawner" />

</launch>